utils/travis/dependencies.sh

```diff
@@ -3,11 +3,14 @@
 
 echo "*****************************************"
 echo "Fetching dependencies"
-echo "Using build system: ${BUILD_SYSTEM}"
+! [ -z ${BUILD_VARIANT} ] && echo "Building ${BUILD_VARIANT}"
+! [ -z ${BUILD_SYSTEM} ] && echo "Using build system: ${BUILD_SYSTEM}"
 echo "*****************************************"
 
-if [[ ${TRAVIS_OS_NAME} == "linux" ]]; then
+if [ ${TRAVIS_OS_NAME} == "linux" ] && ! [ ${BUILD_VARIANT} == "click" ]; then
 	install_linux_deps
+elif [ ${BUILD_VARIANT} == "click" ]; then
+	install_click_deps
```

Comment:
If you set `$PLATFORM` in common.sh, then you could just check the platform and avoid the other checks.